.travis.yml

```diff
@@ -1,10 +1,37 @@
 ---
+language: node_js
 node_js:
-- "0.10"
+  - "0.12"
+
+sudo: false
+
+cache:
+  directories:
+    - node_modules
+
+env:
+  - EMBER_TRY_SCENARIO=default
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-beta
+  - EMBER_TRY_SCENARIO=ember-canary
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-release
```

Comment:
Eventually we are going to want to add this `ember-release` scenario back, but it doesn't surprise me that tests fail for it, since this repo has been neglected for so long.